tests/app/legacy/v2/notifications/test_rest.py

```diff
@@ -136,6 +136,45 @@ async def test_router_returns_400_with_invalid_request_data(
 
         assert response.status_code == status.HTTP_400_BAD_REQUEST
 
+    @pytest.mark.parametrize('route', routes)
+    async def test_auth_error_uses_v2_json_structure(
+        self,
+        client: ENPTestClient,
+        route: str,
+    ) -> None:
+        """Test route can return 201.
+
+        Args:
+            client (ENPTestClient): Custom FastAPI client fixture
+            route (str): Route to test
+
+        """
+        template_id = uuid4()
+        sms_sender_id = uuid4()
```

Comment:
Should be defined in-line below, as they are not used anywhere else in this function and nothing is gained by specifying them here.